hydra-node/json-schemas/api.yaml

```diff
@@ -1430,6 +1437,33 @@ components:
           $ref: "api.yaml#/components/schemas/SequenceNumber"
         timestamp:
           $ref: "api.yaml#/components/schemas/UTCTime"
+    CommitIgnored:
```

Comment:
This message was added so that we are able to figure out when commit gets ignored since it  can't be found in the pending deposits. It is a bit hard to test it out since we would need to be able to either influence the local state of the node or the snapshot.